src/Temporalio/Workflows/WorkflowDefinition.cs

```diff
@@ -104,6 +109,16 @@ private WorkflowDefinition(
         /// </summary>
         public Type[]? FailureExceptionTypes { get; private init; }
 
+        /// <summary>
+        /// Gets the versioning behavior.
+        /// </summary>
+        public VersioningBehavior? VersioningBehavior { get; private init; }
+
+        /// <summary>
+        /// Gets the dynamic options method.
+        /// </summary>
+        public Func<object?, WorkflowDefinitionOptions>? DynamicOptionsMethod { get; private init; }
```

Comment:
Arguably not a "method" like "runMethod", more like a `DynamicOptionsGetter` like "creator", but doesn't really matter